app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+const { displayTopics } = require('./controllers')
+
+
+app.get('/api/topics', displayTopics)
```

Comment:
You might want to consider renaming this, display has connotations of rendering/frontend. I would use something like fetchTopics 👍